assets/src/application/ui/input/EspressoButton/EspressoButton.tsx

```diff
@@ -14,15 +13,15 @@ type ButtonType = React.ComponentType<EspressoButtonProps>;
  * forwardRef to be able to accept
  * onMouseEnter, onMouseLeave, onFocus, onClick events from parent
  */
-const EspressoButton = React.forwardRef<Button, EspressoButtonProps>(
+const EspressoButton = React.forwardRef<typeof Button, EspressoButtonProps>(
```

Comment:
I noticed this pattern in few other cases how is this different than the previous version w/o `typeof`?

Reply:
That's just an extra step

Reply:
ok, but what is the purpose?